huggingface/script/hf-ort.py

```diff
@@ -13,6 +13,14 @@
 from azureml.core import ScriptRunConfig
 from azureml.core.runconfig import PyTorchConfiguration
 
+# NOTE: DeBERTa currently does not run with the Optimum ORT wrapper
+# see: https://github.com/huggingface/optimum/issues/305
+# as well as: https://github.com/microsoft/onnx-converters-private/issues/15
+# DeBERTa code commented out for now
+
+# NOTE: RoBERTa requires ORT==1.12.0
+# see: https://github.com/microsoft/onnxruntime/issues/11268
```

Comment:
Where are we installing ort and setting the version?

Reply:
The Dockerfile base images install the version of ORT based on the CUDA installation & torch version